src/inspector_agent.cc

```diff
@@ -573,6 +585,16 @@ void AgentImpl::InstallInspectorOnProcess() {
   v8::Local<v8::Object> inspector = v8::Object::New(env->isolate());
   READONLY_PROPERTY(process, "inspector", inspector);
   env->SetMethod(inspector, "wrapConsoleCall", InspectorWrapConsoleCall);
+  env->SetMethod(inspector, "addCommandLineAPIMethod", AddAPIMethod);
+
+  v8::Local<v8::Object> console_api = v8::Object::New(env->isolate());
+  env->set_inspector_console_api_object(console_api);
+}
+
+void AgentImpl::AddAPIMethod(const v8::FunctionCallbackInfo<v8::Value>& info) {
+  auto env = Environment::GetCurrent(info);
+  v8::Local<v8::Object> console_api = env->inspector_console_api_object();
```

Comment:
Could you check arguments here and ThrowError if something wrong in the same way like in InspectorWrapConsoleCall?